lib/yard-junk/janitor/text_reporter.rb

```diff
@@ -9,9 +11,20 @@ class TextReporter < BaseReporter
       private
 
       def _stats(**stat)
-        line =
-          '%<errors>i failures, %<problems>i problems (%<duration>s to run)' % stat
-        @io.puts "\n#{line}"
+        @io.puts "\n#{template_for(stat) % stat}"
+      end
+
+      def template_for(stat)
+        if stat[:errors].zero? && stat[:problems].zero?
+          '%<errors>i failures, %<problems>i problems, (%<duration>s to run)'
+        else
+          [
+            Rainbow('%<errors>i failures').red,
+            Rainbow(',').white,
+            Rainbow(' %<problems>i problems').yellow,
+            Rainbow(', (%<duration>s to run)').white
+          ].join('')
+        end
```

Comment:
Don't you think it could be constants? Nothing here (except for selection of "neutral"/"colorized" ouput) depends on instance variables / method arguments.

Reply:
Yes to constants!

Oh, right, schemes of colors would be a useful thing. Would be neat to have already in Rainbow, instead of the using code (this code), but that is a digression.